target/scripts/startup/setup.d/postfix.sh

```diff
@@ -109,29 +109,30 @@ function _setup_postfix_late() {
 function __postfix__setup_override_configuration() {
   __postfix__log 'debug' 'Overriding / adjusting configuration with user-supplied values'
 
+  local OVERRIDE_CONFIG_POSTFIX_MASTER='/tmp/docker-mailserver/postfix-master.cf'
+  if [[ -f ${OVERRIDE_CONFIG_POSTFIX_MASTER} ]]; then
+    while read -r LINE; do
+      [[ ${LINE} =~ ^[0-9a-z] ]] && postconf -P "${LINE}"
+    done < <(_get_valid_lines_from_file "${OVERRIDE_CONFIG_POSTFIX_MASTER}")
+    __postfix__log 'trace' "Adjusted '/etc/postfix/master.cf' according to '${OVERRIDE_CONFIG_POSTFIX_MASTER}'"
+  else
+    __postfix__log 'trace' "No extra Postfix master settings loaded because optional '${OVERRIDE_CONFIG_POSTFIX_MASTER}' was not provided"
+  fi
+
+  # NOTE: Must come after postfix-master.cf as custom parameter support only works provided either config references it.
   local OVERRIDE_CONFIG_POSTFIX_MAIN='/tmp/docker-mailserver/postfix-main.cf'
   if [[ -f ${OVERRIDE_CONFIG_POSTFIX_MAIN} ]]; then
     cat "${OVERRIDE_CONFIG_POSTFIX_MAIN}" >>/etc/postfix/main.cf
     _adjust_mtime_for_postfix_maincf
 
-    # do not directly output to 'main.cf' as this causes a read-write-conflict
+    # Do not directly output to 'main.cf' as this causes a read-write-conflict
     postconf -n >/tmp/postfix-main-new.cf 2>/dev/null
```

Comment:
> Due to postconf -n outputting any errors to /dev/null, this sort of information is omitted from the logs

As long as annoying errors are not expected, `2>/dev/null` should be removed:

```suggestion
    postconf -n >/tmp/postfix-main-new.cf
```

Reply:
> As long as annoying errors are not expected

What do you consider annoying?

I gave an example log line (warning), you get those as well for any duplicate parameters configured IIRC (_due to `postfix-main.cf` appending overrides_).

I can commit your suggestion if you think the added noise is worthwhile? (_[here is why we originally did this](https://github.com/docker-mailserver/docker-mailserver/pull/2598#issuecomment-1140813123)_)

Reply:
Probably I misunderstood something.

By moving the postfix master part at the top, the mentioned error should be fixed? Therefore `postconf -n` should run fine without errors. If that's the case, stderr should not be redirected to dev/null.

Reply:
> By moving the postfix master part at the top, the mentioned error should be fixed?

The log line example shown in this PR would be avoided if the setting is in `master.cf` yes.



> Therefore `postconf -n` should run fine without errors. If that's the case, stderr should not be redirected to dev/null.

See the link to the original reason we added `2>/dev/null` to avoid log spam due to a different condition. I assume that behaviour can still be reproduced and don't see how it would be desirable to show it.